### PR TITLE
fix safari view adaption when auto rotate to portrait

### PR DESCRIPTION
**Pull request by PPpro**
re: https://github.com/cocos-creator/3d-tasks/issues/3986

for details: https://github.com/cocos-creator/engine/pull/6063

### CODE REVIEW

**Comment by pandamicro** [comment]
cocos/core/platform/view.ts:800:78 - error TS2304: Cannot find name 'cc'.

800         if (legacyCC.sys.browserType === legacyCC.sys.BROWSER_TYPE_SAFARI && cc.sys.isMobile) {

CI Failed

File: cocos/core/platform/view.ts [hunk 1]
```diff
@@ -793,6 +793,17 @@ export class View extends EventTarget {
...
         legacyCC.view._orientationChanging = true;
         legacyCC.view._resizeEvent();
+
+        // HACK: show nav bar on iOS safari
+        // safari will enter fullscreen when rotate to landscape
+        // need to exit fullscreen when rotate back to portrait, scrollTo(0, 1) works.
+        if (legacyCC.sys.browserType === legacyCC.sys.BROWSER_TYPE_SAFARI && cc.sys.isMobile) {
+            setTimeout(() => {
+                if (window.innerHeight > window.innerWidth) {
+                    window.scrollTo(0, 1);
```
Comment (pandamicro) [inline cocos/core/platform/view.ts:803]: Can we do this in web adapter level ?
I feel it's dirty to put it here

**Review by pandamicro — CHANGES_REQUESTED** [review]

File: cocos/core/platform/view.ts (hunk 1 above)
Comment (PPpro) [reply cocos/core/platform/view.ts:803]: agree， new issue: https://github.com/cocos-creator/3d-tasks/issues/4025

**Comment by PPpro** [comment]
pr close

**Pull request closed without merging**